### PR TITLE
Impl 821

### PR DESCRIPTION
**Pull request by Huyeng**

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -1686,6 +1710,73 @@ function end_lvl(&$output, $depth) {
...
+    echo '&nbsp; <div class="tooltip tooltip_definition ">';
+        if ($info != "" && $info!=__("Not found", "opendev"))
+          echo '<i class="fa fa-question-circle info-data-classification" title=""></i>';
+        if ($info == "no_evidence_of_adjustment"){
+          echo '<div class="tooltip-info tooltip-no_evidence_of_adjustment">';
+            echo '<p>'.__("ODC is not aware of any adjustments to the concession since it was first granted.", "opendev");
+            echo '</p>';
+          echo '</div>';
+        }else if ($info == "downsized"){
+          echo '<div class="tooltip-info tooltip-downsized">';
```
Comment (acorbi) [inline functions.php:1743]: @Huyeng Have you added this strings to the translation workflow on transiflex? If not, we should in order for the tooltips to be multilingual

**Comment by acorbi** [comment]
@Huyeng On the current test page on PP: https://pp.opendevelopmentmekong.net/wp-admin/post.php?post=1261247&action=edit

The permalink is wrongly pointing to the PROD site: 

screenshot from 2016-04-29 12-37-59

when you click on **View post** then it takes you to that URL on PROD which does not exist yet. Therefore the 404

**Comment by Huyeng** [comment]
Well, this issue doesn't happen for the Profile page, but other as well such post and topic. I wonder this happened before?

File: functions.php (hunk 1 above)
Comment (Huyeng) [inline functions.php:1743]: I pushed the PO file into the language directory. If i'm not wrong the transifex has already pointed the source to file on the github.

**Comment by acorbi** [comment]
@Huyeng Let's tackle the issue separatedly, I have created https://github.com/OpenDevelopmentMekong/opendev-wptheme/issues/830

**Pull request merged**